Src/Optimization/StronginSolver.h

```diff
@@ -0,0 +1,24 @@
+#ifndef STRONGIN_SOLVER_METHOD_H
+#define STRONGIN_SOLVER_METHOD_H
```

Comment:
Do you really need it?

Reply:
This is old style include guard. It is common way to prevent header file duplication.
Now it is more preferable to use `#pragma` once`